### DISCUSSION TITLE
How to get the position of the mouse INSIDE the graph ?

### DISCUSSION THREAD

**Discussion by slechanii** [xyflow/xyflow#1477] [Q&A]
Hi, I need to get the position of the mouse (inside the graph, not relative to the screen) when I right click inside the graph.
I tried using `onPaneContextMenu`   but the coordinates contained inside the event seem to all be relative to the screen and not the graph.

Is there any way currently to get the coordinates of your mouse inside the graph ?

Thanks

**Comment by augusto3691** [comment]
Hi, you achieve this?

**Comment by hankportney** [comment]
I was able to accomplish something similar with `reactFlowInstance.project()`([docs link](https://reactflow.dev/docs/api/react-flow-instance/#project)), which takes a screen position and translates it to a graph position.  

If I understand correctly, you should be able to apply this to `onPaneContextMenu`'s event.
